src/main/java/se/sundsvall/billingdatacollector/integration/db/model/HistoryEntity.java

```diff
@@ -0,0 +1,93 @@
+package se.sundsvall.billingdatacollector.integration.db.model;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import org.hibernate.Length;
+import org.hibernate.annotations.UuidGenerator;
+
+import se.sundsvall.billingdatacollector.integration.db.converter.BillingRecordWrapperConverter;
+import se.sundsvall.billingdatacollector.model.BillingRecordWrapper;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder(setterPrefix = "with")
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(
+	name = "history",
+	indexes = {
+		@Index(name = "idx_family_id", columnList = "family_id"),
+		@Index(name = "idx_flow_instance_id", columnList = "flow_instance_id"),
+	}
+)
+public class HistoryEntity {
+
+	@Id
+	@UuidGenerator
+	@Column(name = "id")
+	private String id;
+
+	@Column(name = "billing_record_wrapper", length = Length.LONG)
+	@Convert(converter = BillingRecordWrapperConverter.class)
+	private BillingRecordWrapper billingRecordWrapper;
+
+	@Column(name = "family_id")
+	private String familyId;
+
+	@Column(name = "flow_instance_id")
+	private String flowInstanceId;
+
+	@Column(name = "created")
+	private LocalDate created;
+
+	@Column(name = "location")
+	private String location;
+
+	@PreUpdate
+	@PrePersist
+	public void prePersist() {
+		if(created == null) {
```

Comment:
Is this needed (if `@PreUpdate` is removed)

Reply:
Nope, it was only there for the ´@PreUpdate´ to not overwrite it, i.e. not needed.